src/base/misc/fatfs.c

```diff
@@ -376,37 +376,35 @@ int read_data(fatfs_t *f, unsigned pos, unsigned char *buf)
 
 static void set_geometry(fatfs_t *f, unsigned char *b)
 {
+  struct on_disk_bpb *bpb = (void *) &b[0x0b];
```

Comment:
Doesn't matter for me, but the rule I myself am
trying to use, is if it is defined as an array - use
indexing. If pointer - use offsetting.
I'll accept any style though.